### DISCUSSION TITLE
question about using 2 ssds for tmp1 and tmp2

### DISCUSSION THREAD

**Discussion by mauroAnnetta** [madMAx43v3r/chia-plotter#464] [Q&A]
hello!
so I have two SSDs NVME, will I be plotting faster with madMax if I use one as tmp1 and the other as tmp2? or will I only divide a bit the degradation of those but speed will be same?

**Comment by 7by24** [comment]
yes , it's faster than single tmp dir.

**Reply by w4d3j** [reply]
I've got two NVME SSDs, different sizes/different brands, and mine are actually faster when configured as a single RAID0 drive than they are as -t and -2.
